controllers/database_controller.go

```diff
@@ -412,20 +491,43 @@ func (r *DatabaseReconciler) reconcilePXC(ctx context.Context, req ctrl.Request,
 	if err != nil {
 		return err
 	}
-	affinity := &pxcv1.PodAffinity{
-		TopologyKey: pointer.ToString(psmdbv1.AffinityOff),
-	}
+
+	pxcSpec := defaultPXCSpec
 	if clusterType == ClusterTypeEKS {
-		affinity.TopologyKey = pointer.ToString("kubernetes.io/hostname")
+		affinity := &pxcv1.PodAffinity{
+			TopologyKey: pointer.ToString("kubernetes.io/hostname"),
+		}
+		pxcSpec.PXC.PodSpec.Affinity = affinity
```

Comment:
We need to set affinity rules for minikube as well

Reply:
I don't think I missed them, my intention was to define them as default here...
https://github.com/percona/dbaas-operator/blob/7c309de83d1e07b9237bb84a50d7ed491d93cf51/controllers/database_controller.go#L81-L83
https://github.com/percona/dbaas-operator/blob/7c309de83d1e07b9237bb84a50d7ed491d93cf51/controllers/database_controller.go#L98-L100
https://github.com/percona/dbaas-operator/blob/7c309de83d1e07b9237bb84a50d7ed491d93cf51/controllers/database_controller.go#L105-L107
... and only overwrite them if in EKS but please double check if I didn't do any mistake.